utils/diagram_generator/lib/main.dart

```diff
@@ -51,11 +51,13 @@ Future<Null> main() async {
   // Add the diagram steps here.
```

Comment:
If overriding the method is not possible for those outlier diagrams, I would put them in a group together here, commented out with a note.